src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/RecalibrationTablesMerger.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.cloud.dataflow.sdk.coders.DefaultCoder;
+import com.google.cloud.dataflow.sdk.coders.SerializableCoder;
+import com.google.cloud.dataflow.sdk.transforms.Combine;
+import org.broadinstitute.hellbender.tools.recalibration.RecalibrationTables;
+
+import java.io.Serializable;
+
+/**
+ * Dataflow-ese for "aggregate RecalibrationTables using their 'combine' method".
+ * <p>
+ * See https://cloud.google.com/dataflow/java-sdk/JavaDoc/com/google/cloud/dataflow/sdk/transforms/Combine.CombineFn
+ * for the Dataflow spec this follows. I've copied their javadoc here.
+ */
+@DefaultCoder(SerializableCoder.class)
+public class RecalibrationTablesMerger extends Combine.CombineFn<RecalibrationTables, RecalibrationTablesMerger.Acc, RecalibrationTables> implements Serializable {
+
+    @DefaultCoder(SerializableCoder.class)
+    public class Acc implements Serializable {
+        RecalibrationTables recalibrationTables;
+
+        public Acc() {
+        }
+    }
+
+    public RecalibrationTablesMerger() {
+    }
+
+    /**
+     * Returns a new, mutable accumulator value, representing the accumulation of zero input values.
+     */
+    @Override
+    public Acc createAccumulator() {
+        return new Acc();
+    }
+
+    /**
+     * Adds the given input value to the given accumulator, returning the new accumulator value.
+     * For efficiency, the input accumulator may be modified and returned.
+     */
+    @Override
+    public RecalibrationTablesMerger.Acc addInput(RecalibrationTablesMerger.Acc accum, RecalibrationTables datum) {
+        if (null == accum.recalibrationTables) {
+            accum.recalibrationTables = datum;
+        } else {
+            accum.recalibrationTables.combine(datum);
+        }
+        return accum;
+    }
+
+    /**
+     * Returns an accumulator representing the accumulation of all the input values accumulated in the merging accumulators.
+     * May modify any of the argument accumulators. May return a fresh accumulator, or may return one of the (modified) argument accumulators.
+     */
+    @Override
+    public RecalibrationTablesMerger.Acc mergeAccumulators(Iterable<Acc> accums) {
+        int x = 0;
```

Comment:
Indeed; please ignore this file. This must be a result of my pipelining multiple pull requests to increase throughput. When merging I'll double-check to make sure this is rebased to the new version of the merger.